DEPLOYING.md

```diff
@@ -8,3 +8,14 @@
 3. Push the new commit and tags to master with `git push origin master --tags`
 4. Create a release on Github. Include changelog in description
 5. Deploy built scripts to s3 via [shipit](https://shipit.shopify.io/shopify/js-buy-sdk/production)
+
+# Updating Documentation
+
+If your change affects how people use the project (i.e. adding or
+changing arguments to a function, adding a new function, changing the
+return value, etc), please ensure the documentation is also updated to
+reflect this. The docs live inside the `docs/` folder and are hosted
+at `http://shopify.github.io/js-buy-sdk`.
```

Comment:
maybe note to say that docs PRs should be separate from code PRs, since the diff will be huge and it's difficult to separate the two for a review?